packages/pangea-sdk/pangea/services/redact.py

```diff
@@ -96,6 +97,7 @@ class StructuredResult(PangeaResponseResult):
     """
 
     redacted_data: Union[Dict, str]  # FIXME: this should be raw json
+    count: int
```

Comment:
oof looks like this class is in flux. not sure if we just want to add docstrings so our sdk site will pick it up